src/app/components/ListSelector/ListSelector.jsx

```diff
@@ -67,11 +66,17 @@ class ListSelector extends React.Component {
           // Leads the user to the 404 page
           this.updateLocation(`${config.baseUrl}404`);
         } else {
+          const data = response.data;
+          const filters = utils.getAllTags(data.picksData.picks);
```

Comment:
Once we get the new data from the server on a client call, we want to update the filters.